dockerclient/client.go

```diff
@@ -872,6 +882,20 @@ func (e *ClientExecutor) CopyContainer(container *docker.Container, excludes []s
 		}
 		return *value, nil
 	}
+	findMissingParents := func(dest string) (parents []string, err error) {
+		destParent := filepath.Clean("/" + dest)
```

Comment:
Isn't this trying to treat `relative path` as an `absolute` one.

After `WORKDIR`  change user could pass a relative path. But `destParent` is cleaning the path and assuming it as an absolute path.

```dockerfile
FROM alpine
RUN mkdir -p test/hello/world

WORKDIR test/hello/world

COPY hello ../hello/hello

WORKDIR ../hello
RUN ls -a

``` 

So `filepath.Clean("/"+"./somedir")` would make it `/somedir`

It would be okay if we are sure that `c.Dest` in `findMissingParents(c.Dest)` is a fully resolved path not `relative`.

Reply:
The callers ensure that it is, so I'll remove the bit that prepends a "/" before cleaning.